core/components/molecules/table/table.js

```diff
@@ -173,6 +176,20 @@ Table.Cell = styled.td`
   width: ${props => props.column.width || 'auto'};
 `
 
+Table.EmptyState = ({ rows, children }) => {
+  if (rows.length > 0 || !children) return null
+
+  const TableEmptyState = styled.div`
+    padding: ${spacing.small};
+    background-color: rgb(250, 250, 250);
+    border-radius: 5px;
```

Comment:
Let's change this to use the common border radius token `$misc.radius`

Reply:
Ohh nice catch, thanks!

Reply:
Done